src/tests/helpers.js

```diff
@@ -0,0 +1,16 @@
+import userEvent from '@testing-library/user-event';
+import { screen } from '@testing-library/react';
+
+export const createTodo = (todo) => {
+    const todoInput = screen.getByRole('textbox');
+    userEvent.type(todoInput, todo);
+
+    const createButton = screen.getByRole('button', {
+        name: /create/i
+    });
```

Comment:
Its a good practise to first check if the btn is enabled or not. This ensured whether the button is in clickable state or not